Orange/canvas/__main__.py

```diff
@@ -344,14 +345,42 @@ def show_message(message):
         def handle_response(result):
             if result == QMessageBox.Yes:
                 success = QDesktopServices.openUrl(
-                    QUrl("http://orange.biolab.si/survey/short.html"));
+                    QUrl("https://orange.biolab.si/survey/short.html"));
                 settings.setValue("startup/show-survey", not success)
             else:
                 settings.setValue("startup/show-survey", result != QMessageBox.No)
 
         question.finished.connect(handle_response)
         question.show()
 
+    check_updates = settings.value('startup/check-updates', True, type=bool)
```

Comment:
Could this code be put in a separate function, called here with settings as parameter?

main is already too long and I am not fond of making it even longer